drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -499,6 +500,11 @@ static int nrf5_tx(const struct device *dev,
 	nrf5_radio->tx_psdu[0] = payload_len + NRF5_FCS_LENGTH;
 	memcpy(nrf5_radio->tx_psdu + 1, payload, payload_len);
 
+#if defined(CONFIG_OPENTHREAD_TIME_SYNC)
+        nrf5_radio->tx_psdu_time_ie_offset = pkt->ieee802154_time_ie_offset;
```

Comment:
The other net_pkt fields have getters such as `net_pkt_ieee802154_ack_fc(pkt)`. Could we add something like that for the time fields?

Reply:
Also, formatting is wrong :)